src/components/safe-apps/SafeAppsErrorBoundary/SafeAppsLoadError.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/router'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+import Button from '@mui/material/Button'
+import OpenInNew from '@mui/icons-material/OpenInNew'
+import { SUPPORT_CHAT_URL } from '@/config/constants'
+
+import css from './styles.module.css'
+
+const SafeAppsLoadError = (): React.ReactElement => {
+  const router = useRouter()
+
+  return (
+    <div className={css.wrapper}>
```

Comment:
Nitpick: I see that the CSS properties here are for the basic layout stuff, and the Material UI has components for layout things: https://mui.com/material-ui/react-grid/ (see layout section in the sidebar)

In my opinion, if we use material UI, we should use material UI 😄 This is not a hard requirement since there's no project-wide consensus for this, just letting you know that such a thing exists

Reply:
yes i know but i don't see a clear path in the project so decided to use the modules as is easy to port the styled-components. For me is ok to use whatever the team  decide if we have a consensus. WDYT @katspaugh ?

Reply:
I would keep it as it is. The less CSS in JSX the better.